plugins/woocommerce/includes/class-woocommerce.php

```diff
@@ -307,6 +310,29 @@ public function add_woocommerce_remote_variant() {
 		}
 	}
 
+	/**
+	 * Set default option values for launch your store task.
+	 */
+	public function add_lys_default_values() {
+		$coming_soon      = current_action() === 'woocommerce_installed' ? 'yes' : 'no';
+		$store_pages_only = WCAdminHelper::is_site_fresh() ? 'yes' : 'no';
+		$private_link     = 'yes';
+		$share_key        = wp_generate_password( 32, false );
+
+		if ( ! get_option( 'woocommerce_coming_soon' ) ) {
+			update_option( 'woocommerce_coming_soon', $coming_soon );
+		}
+		if ( ! get_option( 'woocommerce_store_pages_only' ) ) {
+			update_option( 'woocommerce_store_pages_only', $store_pages_only );
+		}
+		if ( ! get_option( 'woocommerce_private_link' ) ) {
+			update_option( 'woocommerce_private_link', $private_link );
+		}
+		if ( ! get_option( 'woocommerce_share_key' ) ) {
+			update_option( 'woocommerce_share_key', $share_key );
+		}
+	}
+
```

Comment:
Can we add all of these options, including `launch-status` to this list? Otherwise errors occur when using `@woocommerce/data` select to get the values clientside.

https://github.com/woocommerce/woocommerce/blob/64906823664245e6a37391315129c9103ee912cb/plugins/woocommerce/src/Admin/API/Options.php#L157-L226

Reply:
Sure. I'll add theses options to the list!